jobs/kpi-forecasting/kpi_forecasting/metric_hub.py

```diff
@@ -41,6 +48,19 @@ class MetricHub:
     end_date: str = None
     alias: str = None
     project: str = "mozdata"
+    forecast_project: str = None
+    forecast_dataset: str = None
+    forecast_table: str = None
+
+    def fetch(self) -> pd.DataFrame:
+        raise NotImplementedError
+
+@dataclass
+class MetricHub(BaseDataPull):
+    """
+    Programatically get Metric Hub metrics from Big Query.
+    See https://mozilla.github.io/metric-hub/metrics/ for a list of metrics.
```

Comment:
Looks like that URL isn't great...you may want to add a specific product to the end of the URL (e.g., https://mozilla.github.io/metric-hub/metrics/fenix/ )

Reply:
Oh that's goofy, good catch! I'll change it